src/coredump/coredumpctl.c

```diff
@@ -467,6 +519,19 @@ static void analyze_coredump_file(
         *ret_size = UINT64_MAX;
 }
 
+static int get_absolute_filename(sd_journal *j, char **filename) {
+        if (!j->prefix)
+                return 0;
+
+        char *absolute_filename = NULL;
+
+        absolute_filename = path_join(j->prefix, *filename);
+        if (!absolute_filename)
+                return log_oom();
+
+        return free_and_replace(*filename, absolute_filename);
+}
```

Comment:
Use `path_make_absolute()` form path-util.h instead

Reply:
`path_make_absolute` does not work here because `filename` starts with a `/`